src/sdk/models/transfers.ts

```diff
@@ -64,6 +64,11 @@ export enum TransferType {
 	Taxonomy = 'taxonomy'
 }
 
+export enum RenditionType {
```

Comment:
Should we make [naming consistent](https://github.com/portalarchitects/skysync-vnext/blob/bd48a7a2dd008c0d8a0c4b00ae94f8ad49a3f5c2/TransferEngine/src/PortalArchitects.Transfers.Primitives/RenditionSelectionMethod.cs#L9)?